### DISCUSSION TITLE
Hostname support

### DISCUSSION THREAD

**Discussion by 2600i** [pallets/werkzeug#2635] [Q&A]
Is there a way to support a hostname vs just an IP address (e.g. mydomain.app 8080) I'm using an .app domain which is SSL only. I'm having issues with my Cloudflare strict SSL rules.

**Comment by davidism** [comment]
There is nothing special to support here. Werkzeug isn't what controls names, you would need to configure your domain to point to your ip.

**Reply by 2600i** [reply]
I already point my domain to my IP, but Werkzeug won't open up the server as https://mydomain.app:8080, only https://IP:8080 which Cloudflare doesn't like I suspect, but can't confirm... yet.

**Reply by ThiefMaster** [reply]
you must not use the werkzeug dev server in public/production - use a real wsgi server like gunicorn or uwsgi!

**Reply by 2600i** [reply]
Yeah, that was my next thought.  I just spun up a quick test VM for this project and was working through opening a port for django testing, but I will probably just work locally instead.

**Reply by davidism** [reply]
Do not use the development server in production. There is a big red warning about this every time you run it, why did you do so anyway?

Regardless, I can't reproduce your issue, the dev server has no problem binding to a name. You need to fix something in your configuration at a different layer.
